mfr/extensions/tabular/libs/stdlib_tools.py

```diff
@@ -30,3 +34,21 @@ def csv_stdlib(fp):
         })
     rows = [row for row in reader]
     return columns, rows
+
+
+def detect_quoted_fields(dialect, data):
```

Comment:
If this function *is* necessary, its name should indicate that it modifies `dialect`; "detect" implies a read-only action. Perhaps `set_dialect_quotechars`.